CMakeLists.txt

```diff
@@ -139,9 +139,17 @@ add_custom_command(
 
 add_custom_target(phony_always_runs ALL DEPENDS ${CMAKE_BINARY_DIR}/${_script_file})
 
-# todo:: make this generated
+
+# AUTO-GENERATE lanterndb.control file for PostgreSQL
+set(CONTROL_TEMPLATE "${CMAKE_SOURCE_DIR}/cmake/lanterndb.control.template")
+set(CONTROL_OUTPUT "${CMAKE_SOURCE_DIR}/lanterndb.control")
```

Comment:
sorry, just noticed this. Can you put the generated file in the build directory?
'./build' is in gitignore and nothing there will be tracked. but since `cmake` now puts `lanterndb.control` in the source directory, there will be an untracked file under git after running cmake.

Reply:
Makes sense, pushing the change!

Reply:
Good idea, we don't want this file to be showing up in our PRs constantly!